packages/compass-e2e-tests/helpers/commands/add-collection.ts

```diff
@@ -1,17 +1,114 @@
 import type { CompassBrowser } from '../compass-browser';
 import * as Selectors from '../selectors';
 
+type AddCollectionOptions = {
+  capped?: {
+    size: number;
+  };
+  customCollation?: {
+    locale: string;
+    strength: number;
+    caseLevel: boolean;
+    caseFirst: string;
+    numericOrdering: boolean;
+    alternate: string;
+    maxVariable: string;
+    backwards: boolean;
+    normalization: boolean;
+  };
+  timeseries?: {
+    timeField: string;
+    metaField: string;
+    granularity: string;
+    expireAfterSeconds: number;
+  };
+};
+
 export async function addCollection(
   browser: CompassBrowser,
-  collectionName: string
-  // TODO: options for capped collection, use custom collation and time-series
+  collectionName: string,
+  options?: AddCollectionOptions
 ): Promise<void> {
   const createModalElement = await browser.$(Selectors.CreateCollectionModal);
   await createModalElement.waitForDisplayed();
+
   const collectionInput = await browser.$(
     Selectors.CreateDatabaseCollectionName
   );
   await collectionInput.setValue(collectionName);
+
+  if (options && options.capped) {
+    await browser.clickVisible(Selectors.CreateCollectionCappedCheckboxLabel);
+
+    const sizeElement = await browser.$(
+      Selectors.CreateCollectionCappedSizeInput
+    );
+    await sizeElement.waitForDisplayed();
+    await sizeElement.setValue(options.capped.size.toString());
+  }
+
+  if (options && options.customCollation) {
+    await browser.clickVisible(
+      Selectors.CreateCollectionCustomCollationCheckboxLabel
+    );
+
+    for (const [key, value] of Object.entries(options.customCollation)) {
+      await browser.clickVisible(
+        Selectors.createCollectionCustomCollationFieldButton(key)
+      );
+      const menu = await browser.$(
+        Selectors.createCollectionCustomCollationFieldMenu(key)
+      );
+      await menu.waitForDisplayed();
+      const span = await menu.$(`span=${value.toString()}`);
```

Comment:
Had to mix selector strategies to get to the right option:

https://webdriver.io/docs/selectors#element-with-certain-text

https://webdriver.io/docs/api/element/$
> Note: chaining $ and $$ commands only make sense when you use multiple selector strategies. You will otherwise make unnecessary requests that slow down the test (e.g. $('body').$('div') will trigger two request whereas $('body div') does literally the same with just one request)